src/main/resources/com/rultor/agents/req/merge.sh

```diff
@@ -20,6 +20,12 @@ while [ $(git show-branch "${BRANCH}" 2>/dev/null | wc -l) -gt 0 ]; do
 done
 
 git checkout -B "${BRANCH}" "fork/${fork_branch}"
+git fetch origin
+if git log --name-status --oneline HEAD..origin/${head_branch} | grep ".rultor.yml"; then
+  echo "Since original repository has different version of .rultor.yml file you have to sync your fork with the upstream repository."
+  exit 1
+fi
```

Comment:
@xupyprmv no don't overthink this :) No AI needed here. Otherwise I wouldn't have stated this so clearly in the issue description. Using the merge result here is really without alternative, sorry :)

Getting the merge result is actually fairly easy and can be accomplished in two different ways I can think of right now:

* Fetch the merge result from GitHub, see [here](https://help.github.com/articles/checking-out-pull-requests-locally/#modifying-an-inactive-pull-request-locally) 
* **Merge it on the cli side and use the result in Rultor (just cat the result and use it as Shell output)** (This is the one we need to implement, it's 100% safe, because this is what Rultor actually is going to push eventually.)
 
Rultor does not solve any conflicts btw, this needs to be done by the author. All we really need to ensure is that we are actually building against the resulting `.rultor.yml`. This is an easy thing to x